be/src/hash.js

```diff
@@ -0,0 +1,7 @@
+import crypto from 'crypto'
+
+
+export const hash = input => {
+	const hashFunction = crypto.createHash('md5')
```

Comment:
could we move this at the module level?

Reply:
You can't re-use a hashFunction. Realistically we could use a one liner, like

`crypto.createHash('md5').update(input).digest('hex')`

but I thought the above was cleaner/clearer.